### PR TITLE
Fix sda10f urdf

### PR DESCRIPTION
**Pull request by asierfernandez**
We propose a more accurate urdf for sda10f robot. Updated dimensions have been extracted from [3D CAD model](https://www.yaskawa.eu.com/index.php?eID=dumpFile&t=f&f=876&token=f8bae09ca74b0f765920db794a1df3c24acae314) as described in [#232](https://github.com/ros-industrial/motoman/issues/232#issuecomment-468613990)

### CODE REVIEW

File: motoman_sda10f_support/urdf/sda10f_macro.xacro [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
 
-        
-        <xacro:motoman_arm name="${prefix}arm_left" prefix="${prefix}arm_left_" parent="${prefix}torso_link_b1" reflect="1">
-            <origin xyz="0.09996 0.0275 0.32214" rpy="1.57 0 0" />
-        </xacro:motoman_arm>      
-        
-        <xacro:motoman_arm name="${prefix}arm_right" prefix="${prefix}arm_right_" parent="${prefix}torso_link_b1" reflect="-1">
-            <origin xyz="0.09996 -0.0275 0.32214" rpy="1.57 0 0" />
+
+        <xacro:motoman_arm name="${prefix}arm_left" prefix="${prefix}arm_left_" parent="${prefix}torso_link_b1" reflect="1" joint_limited="false">
```
Comment (gavanderhoorn) [inline motoman_sda10f_support/urdf/sda10f_macro.xacro]: Is the `joint_limited` a copy-pasta? It's not an argument supported by the `motoman_arm` macro afaict.

**Comment by gavanderhoorn** [comment]
@asierfernandez: CI seems to fail for `motoman_sda10f_support` with the proposed change.

Can you check locally?

File: motoman_sda10f_support/urdf/sda10f_macro.xacro (hunk 1 above)
Comment (asierfernandez) [reply motoman_sda10f_support/urdf/sda10f_macro.xacro]: sorry I messed up with some other branches

**Comment by asierfernandez** [comment]
I'll check CI next week

**Comment by asierfernandez** [comment]
> @asierfernandez: CI seems to fail for `motoman_sda10f_support` with the proposed change.
> 
> Can you check locally?



> @asierfernandez: CI seems to fail for `motoman_sda10f_support` with the proposed change.
> 
> Can you check locally?

I'll check it next week

**Comment by gavanderhoorn** [comment]
Looks like the `joint_limited` param was the problem.

File: motoman_sda10f_support/urdf/sda10f_macro.xacro (hunk 1 above)
Comment (gavanderhoorn) [reply motoman_sda10f_support/urdf/sda10f_macro.xacro]: So you have a `joint_limited` version of the urdf?

**Comment by gavanderhoorn** [comment]
Getting back to this: the urdf is fixed, but the origins of the meshes are not updated in this PR, which leads to rendering issues (and potential problems with MoveIt's collision detection/avoidance).

Not asking @asierfernandez to fix that after all this time, just making a note about this here.

Example:

rviz_screenshot_2019_08_28-17_24_29

**Comment by asierfernandez** [comment]
We didn't pay attention to meshes, because we were only interested on solving properly the IK.
Meshes should also be updated, but unfortunately we won't work on that in the next months.
@gavanderhoorn  thanks for the comment!

**Comment by gavanderhoorn** [comment]
Just to make sure we don't accidentally merge this in, I'm going to submit a review with *request changes*.

Thanks for the additional info @asierfernandez.

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]
Mesh origins would need to be updated as well.

**Comment by gavanderhoorn** [comment]
We now have #321, which I believe will address the issues raised here as well.

Thanks @asierfernandez for the initial PR :+1:

**Pull request closed without merging**